packages/admin-panel-server/src/viz-builder/extractors/validators.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { yup } from '@tupaia/utils';
+
+import { VisualisationValidator, DashboardVisualisationObject } from '../types';
+
+export class DraftReportValidator implements VisualisationValidator {
```

Comment:
Love the switch to implementing an interface here rather than extending an abstract class 👍